### PR TITLE
Update PDs and skims for Run-3

### PR DESCRIPTION
**Pull request by tvami**
# Replay Request

**Requestor**  

ORM

**Describe the configuration**  
* Release: Unchanged
* Run: 356005 (900b collision), 356069 (cosmics)
* GTs:
   * expressGlobalTag: Unchanged
   * promptrecoGlobalTag: Unchanged
   * alcap0GlobalTag:Unchanged
* Additional changes:

Change PDs and their skims according to Run-3 to be in line with https://github.com/cms-sw/cmssw/pull/38792

**Purpose of the test**  

Test the PDs and then merge to production config
This was partially requested in https://its.cern.ch/jira/browse/CMSTZ-994 and in https://github.com/cms-sw/cmssw/pull/38792 and was discussed at today's joint ops meeting https://docs.google.com/document/d/11zaPV2OpHhaycjM08pL0nTQgOCYxz-CdbpmoOQyD8ok/edit#

**T0 Operations cmsTalk thread**  

https://cms-talk.web.cern.ch/t/replay-and-prod-config-change-to-update-pds-and-skims-for-run-3/13194

### CODE REVIEW

**Comment by tvami** [comment]
atn @sam7k9621

**Comment by tvami** [comment]
check syntax please

**Comment by tvami** [comment]
> atn @sam7k9621

Please let me know if I made any mistakes, thanks!

File: etc/ReplayOfflineConfiguration.py [hunk 1]
```diff
@@ -552,15 +522,26 @@
...
+            "ParkingDoubleMuonLowMass6","ParkingDoubleMuonLowMass7"]
+
 for dataset in DATASETS:
     addDataset(tier0Config, dataset,
                do_reco=True,
                write_dqm=True,
                dqm_sequences=["@common", "@muon", "@heavyFlavor"],
                alca_producers=["TkAlJpsiMuMu", "TkAlUpsilonMuMu"],
-               physics_skims=["LogError", "LogErrorMonitor", "BPHSkim", "MuonPOGJPsiSkim"],
+               physics_skims=["LogError", "LogErrorMonitor", "MuonPOGJPsiSkim"],
```
Comment (sam7k9621) [inline etc/ReplayOfflineConfiguration.py:542]: Same here

**Comment by sam7k9621** [comment]
> > atn @sam7k9621
> 
> Please let me know if I made any mistakes, thanks!

Hi @tvami , I left two comments there, thank you!

**Comment by tvami** [comment]
@sam7k9621 I dont think it got registered, or at least I dont see them. You may need to click to "submit review" or somethng like that

File: etc/ProdOfflineConfiguration.py [hunk 2]
```diff
@@ -588,7 +550,7 @@
                write_dqm=True,
                dqm_sequences=["@common", "@muon", "@heavyFlavor"],
                alca_producers=["TkAlJpsiMuMu", "TkAlUpsilonMuMu"],
-               physics_skims=["LogError", "LogErrorMonitor", "BPHSkim", "MuonPOGJPsiSkim"],
+               physics_skims=["LogError", "LogErrorMonitor", "MuonPOGJPsiSkim"],
```
[the remark this answers is not in the document]
Comment (tvami) [reply etc/ProdOfflineConfiguration.py:553]: But I think you should, no? For the next few weeks these parking datasets will be not parking but will be reco-ed, thus I think you should have the skims there are above...

**Comment by tvami** [comment]
> I left two comments there, thank you!

@sam7k9621 
I spoke to Sal, got convinced you are right, removed the lines in the last push. Thanks for the review if you didnt find anything else to be wrong we'll trigger the replay

**Comment by sam7k9621** [comment]
> > I left two comments there, thank you!
> 
> @sam7k9621 I spoke to Sal, got convinced you are right, removed the lines in the last push. Thanks for the review if you didnt find anything else to be wrong we'll trigger the replay

I am okay with the rest, thank you again!

**Comment by tvami** [comment]
test syntax please

**Comment by tvami** [comment]
@jhonatanamado please trigger a replay when you have some time, thanks!

**Comment by tvami** [comment]
here is the monitoring link
https://monit-grafana.cern.ch/d/t_jr45h7k/cms-tier0-replayid-monitoring?orgId=11&var-Bin=5m&var-ReplayID=220722175739&var-JobType=All&var-WorkflowType=All

**Comment by tvami** [comment]
@sam7k9621 
we get this msg:
```
Message: Can't find physics skim EXODelayedJetMET in CMSSW_12_4_3
```
so this also hasnt propagated back to 12_4_X ??!

**Comment by sam7k9621** [comment]
> @sam7k9621 we get this msg:
> 
> ```
> Message: Can't find physics skim EXODelayedJetMET in CMSSW_12_4_3
> ```
> 
> so this also hasnt propagated back to 12_4_X ??!

Hi, I just noticed that it was only merged to 12_5_X (#37808), sorry for that

**Comment by tvami** [comment]
@jhonatanamado I force-pushed, please trigger a new replay

**Comment by tvami** [comment]
> Hi, I just noticed that it was only merged to 12_5_X ( https://github.com/cms-sw/cmssw/pull/37808 ), sorry for that

I made the backport here:
https://github.com/cms-sw/cmssw/pull/38833

**Comment by tvami** [comment]
New monitoring
https://monit-grafana.cern.ch/d/t_jr45h7k/cms-tier0-replayid-monitoring?orgId=11&var-Bin=5m&var-ReplayID=220722193816&var-JobType=All&var-WorkflowType=All

**Pull request merged**